front50-core/src/main/java/com/netflix/spinnaker/front50/model/Timestamped.java

```diff
@@ -31,4 +33,13 @@
   String getLastModifiedBy();
 
   void setLastModifiedBy(String lastModifiedBy);
+
+  default void setCreatedAt(Long createdAt) {
```

Comment:
Strangely enough, not all objects in `front50` track created timestamps (from the perspective of the API payloads -- we always have it for SQL).